track/views.py

```diff
@@ -48,20 +49,30 @@ def generate_path(prefix, page_id):
         if(prefix == 'en' or prefix == 'fr'):
             return os.path.join(prefix, '{}.html'.format(page_id))
         else:
-            abort(404)
+            abort(HTTPStatus.NOT_FOUND)
+
+    @app.route("/cache-buster")
+    def cache_bust():
+        try:
+            cache.clear()
+            return '', HTTPStatus.NO_CONTENT
+        except Exception as exc:
```

Comment:
I don't know why I wrote this `try`/`except` statement, I'm pretty sure that the only thing this would throw on would be if the cache somehow couldn't be connected to, in which case I don't know what good returning the error to the user with a `500` would be over just letting natural flask exception handler send back the 500 normally since the exception will be logged.

May take this part out